From: REG-Linux <reg@reglinux.org>
Subject: [PATCH] Link libjpeg to the shared library, not just yuvconvert

Upstream libyuv uses JPEG symbols in its shared library but only links
libjpeg to the yuvconvert executable. This causes undefined reference
errors for any consumer linking against libyuv.so.

Signed-off-by: REG-Linux <reg@reglinux.org>
---
 CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -39,6 +39,7 @@
 if (JPEG_FOUND)
   include_directories( ${JPEG_INCLUDE_DIR} )
   target_link_libraries( yuvconvert ${JPEG_LIBRARY} )
+  target_link_libraries( ${ly_lib_shared} ${JPEG_LIBRARY} )
   add_definitions( -DHAVE_JPEG )
 endif()

